src/ontology/hp-edit.owl

```diff
@@ -190951,6 +190953,14 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0041095> "Ar
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0041095> "Decreased middle cerebral artery pulsatility index")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0041095> <http://purl.obolibrary.org/obo/HP_0012022>)
 
+# Class: <http://purl.obolibrary.org/obo/HP_0041096> (Fetal neck mass)
+
+AnnotationAssertion(Annotation(rdfs:comment "PMID:30422714") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0041096> "An abnormal growth (lump) of tissue in the neck region of a fetus. Masses may be simple cystic,, cystic with a few simple septations (fewer than three septations and less than 3-mm thick), cystic with multiple or thick septations (greater than three septations or septations 3-mm-thick or more), a mass that is than is less than 50% solid, and a mass that is 50% solid or more.")
```

Comment:
`Annotation(rdfs:comment "PMID:30422714")` should xref?

Reply:
This goes to show how bad a tool Protege is for editing HPO. The PMIDs are database_cross_references that need to be addded to the definition element.

Reply:
@matentzn We also wanted to use the new Ascertainment terms, could we touch bases about how to to add terms in that subontology?

Reply:
Yes. Can you give one example of a such a term, complex with all metadata annotations, and i will help @LCCarmody to set it up
ASAP! Does that work?